### ISSUE TITLE
CW Keyer for Morse Code over Matrix like CWirc

### ISSUE THREAD

**Issue by MilkManzJourDaddy** [turt2live/matrix-wishlist#243]
CWirc lets users communicate via IRC using Morse Code.  Matrix should have a counterpart.  

If someone joined [#freenode_#cw:matrix.org](https://matrix.to/#/ #freenode_#cw:matrix.org) , they should be able to communicate with their counterpart who joined `irc://irc.freenode.net:6667/cw` .

This will probably depend on PTT, i.e. https://github.com/vector-im/riot-web/issues/5993 .

And there has been a use-case:
http://myspace.voo.be/pcoupard/cw_channel/

**Comment by MilkManzJourDaddy** [comment]
[#freenode_#cw:matrix.org](https://matrix.to/#/#freenode_#cw:matrix.org)

**Comment by MilkManzJourDaddy** [comment]
Yeah, in the Matrix room for this repo' I noticed a print bridge, and there is an issue for a snail mail bridge.  So [I commented that there might as well be a Fax bridge](https://matrix.to/#/!SEKsMhpBOKQFhlpXLR:t2l.io/$15480401641499847TpBUN:matrix.org) and the issue appeared, #232 .

Snail mail came first, then morse code, before the Fax.  So let's not skip any retro facts.

**Comment by 8cH9azbsFifZ** [comment]
Very good idea :)
